pytest_timeouts.py

```
@@ -98,6 +85,18 @@ def parse_timeout(timeout):
         timeout = 0.0 if timeout < 0.0 else timeout
         return timeout
 
+    @staticmethod
+    def configure():
+        TimeoutsPlugin.get_markers_func_name = "get_marker"
+        major_ver, minor_ver, _ = pytest.__version__.split('.')
+        try:
+            major_ver = int(major_ver)
+            minor_ver = int(minor_ver)
+            if (major_ver > 3) or ((major_ver == 3) and (minor_ver >= 6)):
+                TimeoutsPlugin.get_markers_func_name = "iter_markers"
+        except ValueError:
+            pass
```

Why we want ignore error?

---

Code changed